packages/shared/src/utils/digital-options/digital-options.js

```diff
@@ -1,10 +1,12 @@
 export const showDigitalOptionsUnavailableError = (showError, message) => {
     const { title, text, link } = message;
     showError(
-        title,
-        text,
-        link,
-        () => (window.location.href = `${location.protocol}//${location.hostname}/mt5`),
-        false
+        title, // Header
+        text, // Message
+        link, // Redirect label
+        null, // Redirect onClick
+        false, // Should show refresh?
+        '/mt5', // Where to
+        true // With history? (not window.location)
```

Comment:
should we make this into an object so it's clearer what is passed for what?

Reply:
Yes sure.